platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -209,6 +209,94 @@ dependencies {
 ======
 =====
 
+[[deprecate_self_resolving_dependency]]
```

Comment:
❌ This is in the wrong version section now

Reply:
Moved